css-pseudo-4/Overview.bs

```diff
@@ -1634,6 +1634,32 @@ Styling form control pickers: the ''::picker()'' pseudo-element</h3>
 &lt;/style>
   </pre>
 
+<h3 id="select-arrow">
+Select dropdown icon: the ''::select-arrow'' pseudo-element</h3>
+
+  The ''::select-arrow'' pseudo-element only matches when the <a>originating
+  element</a> is a <{select}> element.
+
+  ''::select-arrow'' is a [=part-like pseudo-element=], so there is no
+  restriction on which properties apply to the ''::select-arrow'' pseudo-element.
+
+  ''::select-arrow'' generates a box as if it was an immediate child of its
+  <a>originating element</a>, preceding any boxes generated by the ''::before''
```

Comment:
So I'm wondering if the idea that it precedes the `::before` is incompatible with being part-like.  I'm a little worried that it might be.

Reply:
I made it not part like anymore. Does that resolve this?